vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -132,5 +132,33 @@ void HermesRuntimeHolder::initRuntime() noexcept {
   errorPrototype.setProperty(*m_hermesRuntime, "jsEngine", "hermes");
 }
 
+#ifdef HERMES_ENABLE_DEBUGGER
+
+std::string HermesRuntimeHolder::getDebugTargetName(const DevSettings& devSettings)  noexcept {
+  std::stringstream ss;
+
+  if (!devSettings.debuggerRuntimeName.empty()) {
+    ss << devSettings.debuggerRuntimeName;
+  } else if (!devSettings.debugBundlePath.empty()) {
+    ss << devSettings.debugBundlePath;
+  } else {
+    ss << "Hermes React Native";
+  }
+
+  ss << " (";
+
+  char exeFileName[MAX_PATH];
```

Comment:
if you use `wil::GetModuleFileNameW()` (docs [here](https://github.com/microsoft/wil/wiki/Win32-helpers)) you can avoid the MAX_PATH limitation. not sure if this code has access to wil